Functions/CatalogItems/Get-RsCatalogChildItem.ps1

```diff
@@ -0,0 +1,107 @@
+# Copyright (c) 2016 Microsoft Corporation. All Rights Reserved.
+# Licensed under the MIT License (MIT)
+
+function Get-RsCatalogChildItem
```

Comment:
Alright.
However I'd propose the following aliases:
Get-RsChildItem
rsdir
What's your take on that?